services/core-web/src/components/mine/Permit/ViewPermitOverview.tsx

```diff
@@ -0,0 +1,270 @@
+                </Paragraph>
+              </Col>
+            </Row>
+            <Row>
+              <Col span={24}>
+                <Paragraph strong>Assessed Liability Adjustment</Paragraph>
+                <Paragraph>${latestAmendment.liability_adjustment ?? "-"}</Paragraph>
+              </Col>
+            </Row>
+            <Divider />
+            <Row>
+              <Col span={24}>
+                <Paragraph strong>Tailings Storage Facility Operating Count</Paragraph>
+                <Paragraph>
+                  {
+                    mine.mine_tailings_storage_facilities.filter(
+                      (tsf) => tsf.tsf_operating_status_code === "OPT"
+                    ).length
+                  }
+                </Paragraph>
+              </Col>
+            </Row>
+            <Row>
+              <Col span={24}>
+                <Paragraph strong>Tailings Storage Facility Care and Maintenance Count</Paragraph>
+                <Paragraph>
+                  {
+                    mine.mine_tailings_storage_facilities.filter(
+                      (tsf) => tsf.tsf_operating_status_code === "CAM"
+                    ).length
+                  }
+                </Paragraph>
+              </Col>
+            </Row>
+          </Col>
+          <Col span={12} className="view-permits-detail-section">
+            <Title level={4}>Permit Status</Title>
+            <Row>
+              <Col span={12}>
+                <Paragraph strong>Permit Status</Paragraph>
+                <Paragraph>
+                  {findOptionDescription(
+                    permitStatusOptions,
+                    "permit_status_code",
+                    permit.permit_status_code
+                  )}
+                </Paragraph>
+              </Col>
+              {permit.permit_status_code === "C" && (
+                <Col span={12}>
+                  <Paragraph strong>Date Closed</Paragraph>
+                  <Paragraph>{formatDate(permit?.status_changed_timestamp)}</Paragraph>
+                </Col>
+              )}
+            </Row>
+            <Row>
+              <Col span={24}>
+                <Paragraph strong>Remaining Outstanding Liability Amount (if any)</Paragraph>
+                <Paragraph>{permit.remaining_static_liability ?? "-"}</Paragraph>
+              </Col>
+            </Row>
+            <Row justify="space-between">
+              <Col span={12}>
+                <Paragraph>Updated by: {permit.update_user}</Paragraph>
+              </Col>
+              <Col span={12}>
+                <Paragraph>Updated Date: {formatDate(permit.update_timestamp)}</Paragraph>
+              </Col>
+            </Row>
+            <Title level={4} className="margin-large--top">
+              Permit Documents
+            </Title>
+            <Paragraph>
+              Please upload any documents that support this permit. Documents uploaded here will be
+              viewable by Minespace users.
```

Comment:
this might be a UX question, but are users actually able to upload documents here?

Reply:
Good point.  There is no upload here.